src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs

```diff
@@ -87,7 +87,7 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
                 {
                     _logger.ExecutingAction(actionDescriptor);
 
-                    var startTime = Environment.TickCount;
+                    var startTime = DateTime.UtcNow;
```

Comment:
Yeah, seems like this change is pretty unnecessary. I'll go ahead and close the PR.